styles.css

```diff
@@ -145,3 +152,426 @@ body {
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #344563;
+  padding: 20px 16px 12px 16px;
+}
+
+#tags-list {
+  height: 24px;
+  width: 214px;
+  left: 0;
+  top: 180px;
+  border-radius: 0;
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  text-align: left;
+  padding-left: 8px;
+}
+
+.tag {
+  text-decoration: none;
+  padding: 4px 12px 4px 12px;
+}
+
+#h-tag {
+  height: 24px;
+  width: 57px;
+  left: 0;
+  top: 0;
+  border-radius: 8px;
+  padding: 4px 12px 4px 12px;
+  background: #ebebff;
+  margin-right: 8px;
+}
+
+#c-tag {
+  height: 24px;
+  width: 49px;
+  left: 65px;
+  top: 0;
+  padding: 4px 12px 4px 12px;
+  background: #ebebff;
+  border-radius: 8px;
+  margin-right: 8px;
+}
+
+#j-tag {
+  height: 24px;
+  width: 92px;
+  left: 122px;
+  top: 0;
+  padding: 4px 12px 4px 12px;
+  background: #ebebff;
+  border-radius: 8px;
+}
+
+#html {
+  background: #ebebff;
+  border-radius: 8px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  text-align: left;
+  height: 16px;
+  width: 33px;
+  left: 12px;
+  top: 4px;
+  color: #6070ff;
+}
+
+#css {
+  background: #ebebff;
+  border-radius: 8px;
+  width: 25px;
+  height: 16px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+#JavaScript {
+  background: #ebebff;
+  border-radius: 8px;
+  width: 68px;
+  height: 16px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.actions {
+  height: 72px;
+  width: 295px;
+  left: 0;
+  top: 216px;
+  border-radius: 0;
+  padding: 16px 16px 16px 12px;
+}
+
+.NormalB {
+  height: 48px;
+  width: 126px;
+  left: 0;
+  top: 12px;
+  padding: 12px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 10px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+}
+
+#sbutton {
+  height: 24px;
+  width: 90px;
+  left: 12px;
+  top: 12px;
+  border-radius: nullpx;
+  letter-spacing: 0.03em;
+  text-align: center;
+  color: #396df2;
+  flex: none;
+  order: 0;
+  font-family: 'Poppins', sans-serif;
+  flex-grow: 0;
+  font-size: 14px;
+  font-style: normal;
+}
+
+.Myself {
+  height: 35px;
+  width: 327px;
+  left: 0;
+  top: 0;
+  border-radius: nullpx;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #172b4d;
+  padding-top: 114px;
+  padding-left: 24px;
+  margin: 0 0 0 0;
+}
+
+.social-media-icons-2 {
+  padding: 0;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  height: 20.000062942504883px;
+  width: 170.1666717529297px;
+  left: 24px;
+  top: 470px;
+  border-radius: 0;
+  padding-left: 24px;
+  list-style: none;
+}
+
+.skill {
+  padding: 1px 2px 1px 2px;
+}
+
+.skill-list {
+  list-style: none;
+  padding-left: 24px;
+  padding-right: 24px;
+}
```

Comment:
[OPTIONAL] As mentioned before by the previous reviewer, your about section is not matching the design, specifically by the margins and padding. I think it would be better if you used responsive units as this might fix that. 😉